### DISCUSSION TITLE
Corrections or improvements to the Tatoeba Project

### DISCUSSION THREAD

**Discussion by parfait8566** [Jitendex/Jitendex#23] [Errors and Corrections]
### Thread for corrections or improvement to the Tatoeba Project (example sentences used by Jitendex)

I figured making a new thread for every single issue/improvement might quickly clutter up the discussions page. If you prefer individual posts for clarity, go ahead and close the thread.

**Comment by parfait8566** [comment]
**患う**
JP: 彼女は、幼い頃から喘息を患っている。
EN: She has suffered from asthma since she was little.

**煩う**
JP: あの人は、いつも些細なことで煩っている。
EN: He always worries about little things.

**Reply by stephenmk** [reply]
Are these corrections to sentences that you saw in jitendex, or are you proposing that they be added? I don't see them in the current entry for わずらう.

We can add brand new sentences through Tatoeba, but indexing them to JMdict entries is then a somewhat laborious process. If the currently existing example sentences are adequate, it's really something that I'd rather not do.

**Reply by parfait8566** [reply]
I propose they be added. If it's preferable to keep the existing sentences, I'd change the English translation for 患う to "I've suffered from mastitis before."

**Comment by parfait8566** [comment]
**消息** (improve English translation)
JP: それ以来、彼の消息を聞いていない。
EN: I haven't heard of his whereabouts since then.

**Comment by parfait8566** [comment]
**ずばり** (wrong sense)
The example sentence should be in the second sense

**Comment by parfait8566** [comment]
**一層** (improve English translation)
JP: 今まで以上にいっそう勉強に励みなさい。
EN: You should study even more than before.

**Reply by stephenmk** [reply]
I'm really only interested in improving English sentences in which the translation could cause serious confusion or misunderstanding.

To give an example, I recently changed the meaning for sense 11 of こ【子】. The gloss for this sense is "-er (often of young women)" and it has a note that says "after a noun or -masu stem." I think this is already a pretty confusing definition without more context. But to make matters worse, this was the original example sentence:

> 彼はきっすいのロンドン子です。
> He was born within the sound of the Bow bells. 

This is an incredibly figurative translation. If you don't know what the Bow bells are (probably most people who aren't British), then it makes absolutely no sense. We should hope that the example sentence would at least contain an "-er" to illustrate the meaning of the gloss.

I had the meaning changed to this:

> He's a genuine Londoner.  

Now the "-er" gloss makes sense.

**Reply by parfait8566** [reply]
So if I want more "subtle" changes I'd need to propose them on Tatoeba, correct? Can I help index sentences to JMdict entries?

**Reply by stephenmk** [reply]
Yes, you can make all these changes as long as you have a Tatoeba account. The GUI for making changes is extremely user-unfriendly and creating/editing index strings requires an enormous attention to detail, though. That's why I offer for people to submit corrections here.

There is some documentation regarding the index format here:
https://www.edrdg.org/wiki/index.php/Sentence-Dictionary_Linking

If you go to the "Customize" page on [WWWJDIC](https://www.edrdg.org/cgi-bin/wwwjdic/wwwjdic), there's an option at the bottom of the page to enable links to the Tatoeba indexing GUI. If you enable that option, links to the GUI will appear next to example sentences on WWWJDIC.

Jim Breen (@jimmubreen) has to review all of the edits made to this data each week, so I've started keeping track of my edits in a couple spreadsheets to make his life a little easier. Ideally you'd be able to submit comments along with your edits in the Tatoeba GUI, but this interface was quickly designed for a handful of people to use. It's not incredibly robust.

My recent edits to index values: https://github.com/stephenmk/Jitendex/blob/main/tatoeba_edit_logs/tatoeba_index_edits.csv
Changes to sentence translations: https://github.com/stephenmk/Jitendex/blob/main/tatoeba_edit_logs/tatoeba_meaning_edits.csv

If you don't mind keeping a list of your changes along with some brief comments, I'm sure he'd appreciate it.

**Comment by parfait8566** [comment]
**危ない** (3rd sense)
JP: 危ない空模様だ。

This sentence is not appropriate for the third sense.

**Reply by stephenmk** [reply]
GG5 has a similar sentence ("looks like we're in for a storm") under its "doubtful; questionable" sense of 危ない. That's probably why that Tatoeba sentence ended up in the third sense of this JMdict entry.

I think the JMdict entry for 危ない just needs some improvements in general. Sense 03 could be split into a couple of senses (期待できない vs 信頼できない). I don't think "precarious" should be in sense 01 either.

**Comment by parfait8566** [comment]
**捕まる** (1st sense)
JP: つり革につかまりなさい。電車がすぐ動き出すだろうから。

Sentence in the wrong sense.

**Comment by parfait8566** [comment]
**何たる** (1st sense)
JP: 彼女はもう１０年もフランス語を勉強しているのだから、フランス語の何たるかはよくわかっているはずだ。

Sentence should be in the 2nd sense.

**Comment by arbyste** [comment]
The furikana for 面子 (めんつ) is given in Jitendex in Katakana instead of Hiragana. Is there a reason for that, or is it an error?

**Comment by jimmubreen** [comment]
It's from Chinese and is almost always written in katakana.
See https://www.edrdg.org/jmwsgi/entr.py?svc=jmdict&sid=&q=1533460

Jim


On Fri, 24 Oct 2025 at 08:32, arbyste ***@***.***> wrote:

> The furikana for 面子 (めんつ) is given in Jitendex in Katakana instead of
> Hiragana. Is there a reason for that, or is it an error?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Jitendex/Jitendex/discussions/23#discussioncomment-14766682>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAHFFEM6PSION47O45YRLSL3ZFCN5AVCNFSM6AAAAACKBU25ZSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTINZWGY3DQMQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Jim Breen
https://www.edrdg.org/~jwb/  http://www.jimbreen.org/

**Comment by IanVanLier** [comment]
大鯰 / 大ナマズ should have the furigana おおなまず

**Reply by stephenmk** [reply]
Thanks, I'll fix this.

**Comment by stephenmk** [comment]
Sentence #[10252173](https://tatoeba.org/en/sentences/show/10252173) (priority sentence for やまやま) had its definition ID set to itself in the Tatoeba database. I updated it to the correct corresponding English sentence #[10252179](https://tatoeba.org/en/sentences/show/10252179).

> 欲しいのはやまやまだが予算がない。
> We'd really like one, but we just don't have the money.

I checked all of the other sentences in the database, and there are no other instances of Japanese sentence IDs used as "meaning" IDs like this.

**Comment by IanVanLier** [comment]
光脱毛 should have the furigana ひかりだつもう, I couldn't find a single native pronouncing it as こうだつもう

**Reply by stephenmk** [reply]
Thanks, I submitted an amendment to JMdict for this.
